src/main/resources/swagger/api-docs.yaml

```diff
@@ -137,6 +137,195 @@ paths:
+        - name: configNamespace
+          description: Configuration Namespace
+          required: true
+          type: string
+          in: path
+        - name: configName
+          description: Configuration Name
+          required: true
+          type: string
+          in: path
+        - name: body
+          description: Method Config to Update
+          required: true
+          type: string
+          in: body
+          schema:
+            $ref: ConfigurationIngest
+      responses:
+        200:
+          description: Successful
+        404:
+          description: Workspace or Method Configuration not found
+        500:
+          description: Internal Server Error
+    delete:
+      tags:
+        - Method Configurations
+      operationId: deleteWorkspaceMethodConfig
+      summary: Delete a method configuration in a workspace
+      produces: application/json
+      parameters:
+        - name: workspaceNamespace
+          description: Workspace Namespace
+          required: true
+          type: string
+          in: path
+        - name: workspaceName
+          description: Workspace Name
+          required: true
+          type: string
+          in: path
+        - name: configNamespace
+          description: Configuration Namespace
+          required: true
+          type: string
+          in: path
+        - name: configName
+          description: Configuration Name
+          required: true
+          type: string
+          in: path
+      responses:
+        200:
+          description: Successful request
+        404:
+          description: Workspace or Method Configuration not found
+        500:
+          description: Internal Server Error
+  /workspaces/{workspaceNamespace}/{workspaceName}/method_configs/{configNamespace}/{configName}/rename:
+    post:
+      tags:
+        - Method Configurations
+      operationId: renameWorkspaceMethodConfig
+      summary: Rename a method configuration in a workspace
+      produces: application/json
+      parameters:
+        - name: workspaceNamespace
+          description: Workspace Namespace
+          required: true
+          type: string
+          in: path
+        - name: workspaceName
+          description: Workspace Name
+          required: true
+          type: string
+          in: path
+        - name: configNamespace
+          description: Configuration Namespace
+          required: true
+          type: string
+          in: path
+        - name: configName
+          description: Configuration Name
+          required: true
+          type: string
+          in: path
+        - name: rename
+          description: Method Config Rename
+          required: true
+          type: string
+          in: body
+          schema:
+            $ref: MethodConfigRename
+      responses:
+        200:
+          description: Successful request
+        404:
+          description: Workspace or Method Configuration not found
+        500:
```

Comment:
rawls API documents it can return a 409	Method config by that name already exists in workspace